specs/schemas/workspace.json

```diff
@@ -1,10 +1,14 @@
 {
-  "$schema": "<http://json-schema.org/draft-07/schema#>",
+  "$schema": "http://json-schema.org/draft-07/schema",
   "type": "object",
   "properties": {
     "version": {
       "type": "string"
     },
+    "kiotaVersion": {
```

Comment:
Looks like it's optional. In that one can pass an empty string. Is there a way to specify this in the schema?

Reply:
It's optional in this schema.